app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -134,14 +134,21 @@ const AvailableActions = createReactClass({
       }
     // If user has no role or is logged out
     } else if (!course.ended) {
-      controls.push(
-        <div key="join" className="available-action"><button onClick={this.join} className="button">{CourseUtils.i18n('join_course', course.string_prefix)}</button></div>
-      );
-      // On P&E Dashboard, offer option to join as online volunteer
-      if (!Features.wikiEd && course.online_volunteers_enabled) {
+
+      //If it's enrollment page, don't show 'Join course' button in Actions to avoid confusion
+      const urlParams = new URLSearchParams(window.location.search);
+      const isEnrollmentURL = urlParams.has('enroll');
```

Comment:
This if-if is unnecessary, and there is a lot of those already. I would recommend declaring variables before if-else statement, so we could have just one if and one condition.

Reply:
Hi @1v4n4. Thanks for the recommendation. I have tried to update it accordingly with new commit. Do let me know if it's okay.

Reply:
It looks good to me!